packages/react-examples/.storybook/preview.js

```diff
@@ -25,11 +25,16 @@ if (
     'react-button',
     'react-image',
     'react-link',
+    'react-accordion',
```

Comment:
we (me and @bsunderhus) run into runtime SB runtime errors today and we're able to figure it out and fix it.

> yarn workspace @fluentui/react-accordion start

![image](https://user-images.githubusercontent.com/1223799/110322866-0f571a80-8014-11eb-8996-086dcedf9782.png)


**issues:**
#### 1. duplicating decorators when using old api
As was mentioned within [migration to SB v6, decorators are duplicated, which leads to unstable behaviours](https://github.com/microsoft/fluentui/pull/17033#issuecomment-782287764). For that reason I'd leverage `Set` as a temporary workaround for how we resolve magic strings and webpack source evaluation.

```js
// @ts-check

/**
* @file packages/react-examples/.storybook/preview.js
*/

/**
 * @type {Set<import('@storybook/react').DecoratorFn>}
 */
const packageDecorators = new Set();

function registerAdditionalDecorators(decorators){
  decorators.forEach(decorator => addDecorator(decorator));
}

if (
  [...packageName].includes(
    'PACKAGE_NAME',
  )
) {
  packageDecorators
    .add(withStrictMode);
}

if ([...packageNames].includes('PACKAGE_NAME')) {
  packageDecorators.add(withFluentProvider).add(withStrictMode);
}

registerAdditionalDecorators(packageDecorators)
```

#### 2. withCompatThemeProvider doesn't exist anymore
This was the main culprit, an old good calling `undefined` :) , so the fix was to remove this. I dunno when this happened but for now, I'd suggest to add runtime check to mitigate such a blockers (which will be no longer present after [TS path aliases is implemented](https://github.com/microsoft/fluentui/issues/16889))

```
// contrived example
function validateFluentuiStorybookApis(decorators){
   decorators.forEach(decorator=>{
     if(decorator == null || typeof decorator !== 'function'){
        throw new Error(`one of @fluentui/storybook decorators doesnt exist`)
     }
})
}
```

Reply:
addressed in https://github.com/microsoft/fluentui/pull/17335